pandas/core/nanops.py

```diff
@@ -261,19 +266,24 @@ def _na_ok_dtype(dtype):
 
 def _view_if_needed(values):
     if is_datetime_or_timedelta_dtype(values):
-        return values.view(np.int64)
+        try:
```

Comment:
then let's add view rather than do this

Reply:
also waiting on #24024.

Reply:
well, then this is going to have to wait for .view unless you do it here.

Reply:
#24024 has one, but I don't know if it'll stick around. I'm pretty strongly +1 on keeping it though.